proposed/0006-Launcher.md

```diff
@@ -0,0 +1,204 @@
+- Feature Name: SAFE-Launcher
+- Type: New Product
+- Related components: [safe_client](https://github.com/maidsafe/safe_client), [safe_nfs](https://github.com/maidsafe/safe_nfs)
+- Start Date: 11-September-2015
+
+# Summary
+
+Launcher will be a gateway for any App that wants to work on the SAFE-Network on a user's behalf.
```

Comment:
This needs fleshed out a bit.  It should still be a single paragraph, but it should allow third-party readers to get a rough idea of what the product will be, and this also should allow readers to disambiguate between the other proposed Launcher RFC.